### PR TITLE
[Docs] Fixing the page layout CSS

### LINKED ISSUE #5113: [Docs] Page layout includes unnecessary margin
#### Current State
Docs pages that use either the `default` or `page` [layout](https://github.com/meshery/meshery/blob/master/docs/_layouts/) seem to have an unnecessary margin included, wasting space that can be used for docs text.

Screen Shot 2022-02-23 at 10 47 27 AM

#### Desired State
Recover the unused whitespace; eliminate the margin/padding in screenshot.

---

#### [Contributor Guides](https://docs.meshery.io/project/contributing) and Resources
- 📚 [Instructions for contributing to documentation](https://docs.meshery.io/project/contributing/contributing-docs)
   - Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
- 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/build-and-release)
- 🎨 Wireframes and designs for Meshery UI in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI)
- 🙋🏾🙋🏼 Questions: [Layer5 Discussion Forum](https://discuss.layer5.io) and [Layer5 Community Slack](http://slack.layer5.io)

### PR DESCRIPTION
**Pull request by sidd-oo**
Signed-off-by: sidd-oo <sahoosiddharth3@gmail.com>

**Description**
Docs pages had max width of 80% for `p`, `ul`, `li` tags, it was wasting space that can be used for docs text. So I removed the max-width for the above-mentioned tags.

This PR fixes #5113 

**Notes for Reviewers**
All the changes are done in the `main.css` file which resides inside `docs/assets/css/main.css`

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [X] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 753fb8ed9994ec8130d016d662ddf04bbd40f2d4 at: https://6218a645529a4b82515aa1aa--meshery-docs-preview.netlify.app

**Review by Abhijay007 — DISMISSED** [review]
LGTM !! as it is responsive and applied to other pages as well which have the same issue of extra white space.

**Review by alphaX86 — APPROVED** [review]
LGTM on my side 👍 🎉

**Review by warunicorn19 — CHANGES_REQUESTED** [review]
Just a small nitpick, could you please make this right sidebar full length. 
Screenshot 2022-02-25 at 10 20 24 AM

File: docs/assets/css/main.css [hunk 1]
```diff
@@ -15556,9 +15556,6 @@ a.text-dark:focus {
     .td-content>.lead,
     .td-content>h1,
     .td-content>h2,
-    .td-content>ul,
```
Comment (alphaX86) [inline docs/assets/css/main.css:15559]: Maybe... adding this back would do the trick for restoring right sidebar?

Comment (sidd-oo) [reply docs/assets/css/main.css:15559]: No, I tried it didn't work. @alphaX86

**Comment by l5io** [comment]
🚀 Preview for commit e21c3e30697cd28e7858d04be06d4f6dea166b84 at: https://6219b9d7a0e07549ff5258d0--meshery-docs-preview.netlify.app

**Comment by sidd-oo** [comment]
Kindly check the new changes to fix the height of the Table of content @alphaX86 @Abhijay007

**Review by alphaX86 — APPROVED** [review]
Ok, it worked. Right TOC bar is fixed

**Review by Abhijay007 — APPROVED** [review]
LGTM 👍

**Review by warunicorn19 — APPROVED** [review]
Looks good, Thank you @sidd-oo

**Pull request merged**